src/App/Pages/Accounts/LoginSsoPageViewModel.cs

```diff
@@ -268,18 +276,14 @@ private async Task LogIn(string code, string codeVerifier, string orgId)
                 // In the standard, non TDE case, a user must set password if they don't
                 // have one and they aren't using key connector.
                 // Note: TDE & Key connector are mutually exclusive org config options.
-                if (response.ResetMasterPassword || (decryptOptions?.RequireSetPassword ?? false))
+                if (response.ResetMasterPassword || (decryptOptions?.RequireSetPassword == true))
                 {
+                    // TODO: We need to look into how to handle this when Org removes TDE
+                    // Will we have the User Key by now to set a new password?
                     StartSetPasswordAction?.Invoke();
                     return;
                 }
 
-                if (response.ForcePasswordReset)
-                {
-                    UpdateTempPasswordAction?.Invoke();
-                    return;
-                }
```

Comment:
🤔 Is this ok to be removed for a user that belongs to an Org without TDE enabled? I think it'll be shown later at the main page given the ForcePasswordResetReason but wanted to ask just in case

Reply:
Yup that is exactly the case.